Automattic/newspack-plugin
# fix(ras): update transactional email logo default size

### All Submissions:






### Changes proposed in this Pull Request:



Closes https://app.asana.com/0/1207817176293825/1208629737155849/f

This PR adjusts the default site logo width in transactional emails to the expected value ([here in the newsletters plugin](https://github.com/Automattic/newspack-newsletters/blob/61d908900a016ddd66d63c9dc9f3230351ef077f/includes/class-newspack-newsletters-renderer.php#L580))

### How to test the changes in this Pull Request:

1. As admin, go to Newspack > Engagement > Show Advanced Settings > Transactional Email Content
2. Reset the Set a New Password template
3. Edit the template and confirm the logo is 125px wide
4. Send a test email and confirm the logo is 125px wide in the sent email


### Other information:
